app/src/sharedTest/java/org/oppia/android/app/profile/AddProfileActivityTest.kt

```diff
@@ -983,11 +984,11 @@ class AddProfileActivityTest {
         closeSoftKeyboard()
       )
       testCoroutineDispatchers.runCurrent()
-      onView(withId(R.id.add_profile_activity_allow_download_switch))
+      onView(withId(R.id.add_profile_activity_allow_download_constraint_layout))
```

Comment:
Few other test cases using switch. 
 - `testAddProfileActivity_configChange_inputPin_allowDownloadIsEnabled`
 - `testAddProfileActivity_inputPin_configChange_downloadAccessSwitchIsOn`

Reply:
Looks like `testAddProfileActivity_inputPin_configChange_downloadAccessSwitchIsOn` is passing although we make the clickable false for switch. 

Could you please check if this test case is passing on espresso?

Reply:
`testAddProfileActivity_configChange_inputPin_allowDownloadIsEnabled ` and 
`testAddProfileActivity_inputPin_configChange_downloadAccessSwitchIsOn `
tests are correct because here we intentionally want to check if `Switch` is enabled or not.

Reply:
For `testAddProfileActivity_inputPin_configChange_downloadAccessSwitchIsOn `
Updated the click listener line to `onView(withId(R.id.add_profile_activity_allow_download_constraint_layout)).perform(click())`